### PR TITLE
Clean PythonRange printing

### PR DESCRIPTION
**Pull request by EmilyBourne**
As suggested in #1643 , don't print the step of a range if that step is 1.

### CODE REVIEW

**Review by bauom — APPROVED** [review]
everything looks good to me.

File: docs/quickstart.md [hunk 1]
```diff
@@ -289,14 +289,14 @@ def matmul(a: 'float[:,:](order=C)',
...
-#$ omp for schedule(runtime)
+    #$ omp parallel
+    #$ omp for schedule(runtime)
     for i in range(m):
         for j in range(n):
             c[i, j] = 0.0
             for k in range(p):
                 c[i, j] += a[i, k] * b[k, j]
-#$ omp end parallel
+    #$ omp end parallel
```
Comment (yguclu) [inline docs/quickstart.md:300]: Are we 100% sure that the leading white spaces are ignored?

**Review by yguclu — APPROVED** [review]

File: docs/quickstart.md (hunk 1 above)
Comment (EmilyBourne) [reply docs/quickstart.md:300]: The new version is translated to:
```fortran
  function matmul(a, b, c) result(Out_0001)

    implicit none

    integer(i64) :: Out_0001
    real(f64), intent(in) :: a(0:,0:)
    real(f64), intent(in) :: b(0:,0:)
    real(f64), intent(inout) :: c(0:,0:)
    integer(i64) :: m
    integer(i64) :: p
    integer(i64) :: q
    integer(i64) :: n
    integer(i64) :: r
    integer(i64) :: s
    integer(i64) :: i
    integer(i64) :: j
    integer(i64) :: k

    m = size(a, 2_i64, i64)
    p = size(a, 1_i64, i64)
    q = size(b, 1_i64, i64)
    n = size(b, 2_i64, i64)
    r = size(c, 2_i64, i64)
    s = size(c, 1_i64, i64)
    if (p /= q .or. m /= r .or. n /= s) then
      Out_0001 = -1_i64
      return
    end if
    !$omp parallel
    !$omp do schedule(runtime)
    do i = 0_i64, m - 1_i64, 1_i64
      do j = 0_i64, n - 1_i64, 1_i64
        c(j, i) = 0.0_f64
        do k = 0_i64, p - 1_i64, 1_i64
          c(j, i) = c(j, i) + a(k, i) * b(k, j)
        end do
      end do
    end do
    !$omp end do
    !$omp end parallel
    Out_0001 = 0_i64
    return

  end function matmul
```

So there don't seem to be any problems. I would be more worried about wrong indenting causing problems in Python

**Review by yguclu — APPROVED** [review]

**Pull request merged**